/usb_camera/usb_camera_driver:
  ros__parameters:
    autostart: true
    base_topic_name: camera
    best_effort_qos: true
    brightness: 0.0
    camera_calibration_file: file:///home/neo/workspace/src/usb_camera_driver/config/camera.yaml
    camera_id: 0
    camera_device_file: /dev/video0
    camera_name: camera
    exposure: 0.0
    fps: 20
    frame_id: map
    image_height: 480
    image_width: 640
    publisher_depth: 5
    rotation: 0
    wb_temperature: 0.0
    # image_transport parameters
    usb_camera_driver:
      camera:
        image_color:
          format: jpeg
          jpeg_quality: 95
          png_level: 3
        image_rect_color:
          format: jpeg
          jpeg_quality: 95
          png_level: 3